codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/validation/UnaryFunctionCall.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ package software.amazon.smithy.aws.typescript.codegen.validation;
+
+import java.util.regex.Pattern;
+
+/**
+ * For handling expressions that may be unary function calls.
+ */
+public abstract class UnaryFunctionCall {
```

Comment:
Why is this class `abstract`?

Reply:
Carry over from the original addition to smithy-typescript: https://github.com/awslabs/smithy-typescript/blob/main/smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/validation/UnaryFunctionCall.java. It doesn't need to be abstract.

Reply:
there might've been a linter warning when a class only has static methods and isn't abstract